adops/setting-up-prebid-native-in-dfp.md

```diff
@@ -21,34 +21,33 @@ For more information about DFP native ad setup, see the [DFP native ads document
 * TOC
 {:toc}
 
-## 1. Add a native ad unit
+## 1. Create a native ad unit
 
-You can specify a size for the ad unit, or specify the "fluid" size.  In this case we'll go with **Fluid**.
+Under **Display > Native**, select the **HTML & CSS editor** option.
 
-{: .pb-img.pb-md-img :}
-![prebid native adunit]({{site.github.url}}/assets/images/ad-ops/dfp-native/prebid_native_adunit.png)
+## 2. Define ad settings
 
-## 2. Create a custom native ad format
+For **Ad size** you can specify a specific size for the ad unit or specify the "fluid" size.  In this case we'll go with **Fluid**.
 
-For Prebid, create a [custom native ad format](https://support.google.com/dfp_sb/answer/6366911?hl=en).
+Select **New format** under **Custom format**. (If you've already created an ad unit with the format you want, you can select **Existing format** and select the format to apply to this ad unit.)
 
-Make sure to remove all variables except **Click-through URL**.
+Click **Add Variable**. Use the **File** type and create a **ClickthroughURL** variable.
 
 {: .pb-img.pb-md-img :}
-![prebid native format]({{site.github.url}}/assets/images/ad-ops/dfp-native/prebid-native-format.png)
+![native adunit settings]({{site.github.url}}/assets/images/ad-ops/dfp-native/prebid_native_settings.png)
 
-## 3. Create new native style
+## 3. Style your native ad
 
-In this example, we're targeting `"prebid_native_adunit"`, so the *Size* is set to **Fluid**.
+You can add HTML and CSS to define your native ad template. To allow for native impression trackers and click trackers within a Prebid native creative template, you'll need to include a CDN-hosted script in the HTML, as shown here (see Example HTML below for the full script):
 
-Add the HTML and CSS that define your native ad template. Note that `%%PATTERN%%` macros can be included in either field, and the HTML can contain JavaScript.  For more information, see the [DFP native styles docs](https://support.google.com/dfp_premium/answer/6366914).
+{: .pb-img.pb-md-img :}
+![native ad styling]({{site.github.url}}/assets/images/ad-ops/dfp-native/prebid_native_styling.png)
 
 {: .alert.alert-danger :}
 **Native impression and click tracking requirements**  
-You must include the `postMessage` and `onclick` JavaScript snippets as shown in the example code below so that impression trackers and click trackers will fire.
+Any link that should fire a click tracker needs to include a `data-adid` attribute set to `hb_adid`. This attribute is required because the script needs the bidId/adId, which can be filled in only by the targeting key on the ad server, not from within the script.
```

Comment:
I'm not sure I understand this.  Should the snippet below have a 'data-adid' attribute, because I'm not seeing one present?

Reply:
The `data-adid` in the blurb above should be changed to `pbAdId`; this is the attribute in the anchor tag that stores the value from `hb_adid` and is read by the new script.